en/docs/developer-portal/manage-application.md

```diff
@@ -51,3 +51,62 @@ Let's get started!
 8. Click **:// CURL** to copy the cURL command template for generating the access token. You can add the consumer key and consumer secret to the URL and invoke the token endpoint to generate an API access token.
 
     ![cURL command](../assets/img/developer-portal/manage-applications/curl-command.png){.cInlineImage-half}
+
+
+## Revoke access tokens
+
+In the event of a theft, security violation, or a precaution, an admin can revoke an access token. Choreo provides you with a revoke token endpoint. You can use a utility like a cURL to invoke this endpoint and revoke your access token.  
+
+1. Sign in to Choreo Developer Portal at [https://devportal.choreo.dev](https://devportal.choreo.dev). Alternatively, click the Developer Portal link on the Choreo console header. 
+
+    ![Developer Portal](../assets/img/developer-portal/manage-applications/developer-portal.png){.cInlineImage-half}
+
+2. Click **Applications**.
+3. Click **Production Keys** -> **OAuth 2.0 Tokens** on the left panel.
+4. You will find the **Consumer Key** (client ID), **Consumer Secret** (client secret), **Token Endpoint**, and **Revoke Endpoint** listed here. You can use these values to revoke the access token. 
```

Comment:
Since this step just provides info and does not really mention an action to perform, should we remove the numbering?

Reply:
done